app/upgrades/v24/upgrades_test.go

```diff
@@ -100,6 +100,12 @@ func (s *UpgradeTestSuite) TestUpgrade() {
 	s.Require().NoError(err)
 	s.Require().Equal(protorevtypes.DefaultBaseDenoms, newBaseDenoms)
 
+	// check that old lockable durations were not single param
+	oldLockableDurations := s.App.IncentivesKeeper.GetLockableDurations(s.Ctx)
+	s.Require().Equal(len(oldLockableDurations), 4)
+	oldLockableDurations = s.App.IncentivesKeeper.GetLockableDurations(s.Ctx)
```

Comment:
I think that this might have been intended to be pool incentives. Currently, the check is repeated on the incentives module twice

Reply:
Has been solved in the most recent commit